vendor/github.com/docker/go-connections/tlsconfig/certpool_other.go

```diff
@@ -4,7 +4,6 @@ package tlsconfig
 
 import (
 	"crypto/x509"
```

Comment:
How did these vendor changes slip in here? Also see below.

Reply:
Fixed via a new squashed commit. Something weird was going on with git, but squashing fixed it. I couldn't remove them in subsequent commits weirdly.